app/src/main/java/deakin/gopher/guardian/services/EmailPasswordAuthService.kt

```diff
@@ -54,22 +49,6 @@ class EmailPasswordAuthService(
         }
     }
 
-    fun updatePassword(newPassword: String): Task<Void>? {
-        val currentUser = auth.currentUser
-        return currentUser?.let { user ->
-            user.updatePassword(newPassword).addOnCompleteListener { task ->
-                if (task.isSuccessful) {
-                    // Password update successful, update the timestamp
-                    userDataService.updateLastPasswordChangeTimestamp(user.uid, System.currentTimeMillis())
-                    Log.v(logTag, "Password updated successfully for User ${user.uid}")
-                } else {
-                    // Failed to update password
-                    Log.e(logTag, "Failed to update password: ${task.exception}")
-                }
-            }
-        }
-    }
-
```

Comment:
Is there a reason why we are getting rid of the `updatePassword` function?

Reply:
currently the faulty codes have been removed before the faulty commit, This was causing issue on the app side not being able to run on user side.

Reply:
Okay, approving as this entire logic will be migrating to a backend API.